velox/connectors/hive/PartitionIdGenerator.h

```diff
@@ -46,18 +49,37 @@ class PartitionIdGenerator {
  private:
   static constexpr const int32_t kHasherReservePct = 20;
 
+  // Computes value IDs using VectorHashers for all rows in 'input'.
+  void computeValueIds(
+      const RowVectorPtr& input,
+      raw_vector<uint64_t>& valueIds);
+
+  // Updates value IDs in 'partitionIds_' mapping in case of rehash.
```

Comment:
Consider to add a note here
```
//
// NOTE: in case of rehash, we will update value id but keep its partition id. The function rebuilds 'partitionIds_' by re-calculating the value ids with updated 'hashers_'.
```